packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -498,6 +496,7 @@ const renderer =
                         skipFirstNotification: true
                     }),
 
+                    // TODO: investigate why there are 3 similar instances of this CopyPlugin in this file
```

Comment:
I think we had a bad merge that caused this 😞

Reply:
No, this was always this way. `webpack/config.js` is a centralized place for storing **FIVE** different webpack configs.  Looking over the code there are `client` / `clientOptional` / `renderer` / `ssr` / `requestProcessor` webpack configs

All of these call `baseConfig(<target>)` where `target` is an enum with valid range `'node' || 'web'` the function is not properly obfuscated with something like `baseConfig(<target>, <type>)` where type would be an enum of range `client || clientOptional || renderer || ssr || requestProcessor`

So, this is how things have always worked, no need to put ourselves at risk by refactoring this now, fixing it would be a non-breaking change we can release in any future version

Reply:
(pushed a commit to resolve)